libr/util/format.c

```diff
@@ -542,16 +547,23 @@ static void r_print_format_time(const RPrint* p, int endian, int mode,
 		}
 		free (timestr);
 	} else if (MUSTSEEJSON || MUSTSEESTRUCT) {
-		char *timestr = strdup (asctime (gmtime_r ((time_t*)&addr, &timestruct)));
+
```

Comment:
why did you add these random newlines in the last commit?

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
@radare https://github.com/radareorg/radare2/pull/17465#discussion_r483578503

Reply:
Oh my god, the web application seems to have bugged, sorry for the ping